### DISCUSSION TITLE
Matplotlib

### DISCUSSION THREAD

**Discussion by tgederberg** [Homebrew/discussions#3985] [Everyday usage] [answered]
I'm surprised that matplotlib is not available via Homebrew.  Is there a reason is it not available.

**Comment by p-linnane** [comment, marked as answer]
It was removed a number of years back. See: https://github.com/Homebrew/homebrew-cask/pull/13573.

If you'd like to try to readd it, I will review a PR.

**Reply by tgederberg** [reply]
I'm brand new using Homebrew (have been using MacPorts for years but decided to give Homebrew a try on my MacBook Pro).  At this point, I'm not sure how I would add it.  Thanks.

**Reply by p-linnane** [reply]
Here's a helpful link to get you started: https://docs.brew.sh/Adding-Software-to-Homebrew

**Comment by tgederberg** [comment]
After reading some of the documentation, it was fairly easy to install via pip3.

brew install python@3.10
pip3 install numpy
pip3 install scipy
pip3 install matplotlib
